packages/animations/test/fade_scale_transition_test.dart

```diff
@@ -146,27 +156,35 @@ void main() {
       // No scale animations on exit transition.
       double topFadeTransitionOpacity = _getOpacity(key, tester);
       double topScale = _getScale(key, tester);
+      Alignment topScaleAlignment = _getScaleAlignment(key, tester);
       expect(topFadeTransitionOpacity, 1.0);
       expect(topScale, 1.0);
+      expect(topScaleAlignment, Alignment.center);
 
       await tester.pump(const Duration(milliseconds: 25));
       topFadeTransitionOpacity = _getOpacity(key, tester);
       topScale = _getScale(key, tester);
+      topScaleAlignment = _getScaleAlignment(key, tester);
       expect(topFadeTransitionOpacity, closeTo(0.66, 0.05));
       expect(topScale, 1.0);
+      expect(topScaleAlignment, Alignment.center);
 
       await tester.pump(const Duration(milliseconds: 25));
       topFadeTransitionOpacity = _getOpacity(key, tester);
       topScale = _getScale(key, tester);
+      topScaleAlignment = _getScaleAlignment(key, tester);
       expect(topFadeTransitionOpacity, closeTo(0.33, 0.05));
       expect(topScale, 1.0);
+      expect(topScaleAlignment, Alignment.center);
 
       // End of opacity animation
       await tester.pump(const Duration(milliseconds: 25));
       topFadeTransitionOpacity = _getOpacity(key, tester);
       expect(topFadeTransitionOpacity, 0.0);
       topScale = _getScale(key, tester);
       expect(topScale, 1.0);
+      topScaleAlignment = _getScaleAlignment(key, tester);
+      expect(topScaleAlignment, Alignment.center);
```

Comment:
All tests seem to be using `Alignment.center`. There should also be one that doesn't use the default.

Reply:
so in order to modify `FadeScaleTransitionConfiguration`, I need to extend it and override `transitionBuilder` with different `FadeScaleTransition.alignment`. Is it okay to put the extended configuration in that test?

Reply:
I think that's, unfortunately, the only way we can have FadeScaleTransition properly make use of the alignment parameter. I'm just worried about its ease-of-use since the user would have to extend the `FadeScaleTransitionConfiguration` widget, as you mentioned. 

I still think it's okay to do it, but we'll probably have to add something in the docs so that users are aware how to achieve this effect.

Reply:
I was thinking if I would add `alignment`/`scaleAlignment` param for `FadeScaleTransitionConfiguration`  to avoid extending it but since the motion system docs only shows dialog/modal scales at center, I think it doesn't need that param, even some docs on achieving that effect on dialogs/modals.

I just asking because I want to follow the same test schemes on `fade_scale_test.dart`, using a modal to test `FadeScaleTransition`.